src/style.scss

```diff
@@ -108,6 +113,10 @@ textarea {
   color: white !important;
 }
 
+.separator {
```

Comment:
We don't need to create a new class for this, you can use the class from PrimerCSS `d-inline-block` directly on the with

Reply:
preparing for styling the separator